src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -887,7 +887,7 @@ void LFUDAPolicy::cleaning(const DoutPrefixProvider* dpp)
 	      }
 	    } //end-if (block.version == entry->version)
 	  } //end - else if op_ret == 0
-	  ldpp_dout(dpp, 10) << "D4NFilterObject::" << __func__ << "(): Removing object name: "<< c_obj->get_name() << " score: " << std::setprecision(std::numeric_limits<double>::max_digits10) << e->creationTime << " from ordered set" << dendl;
+	  ldpp_dout(dpp, 10) << "D4NFilterObject::" << __func__ << "(): Removing object name: "<< c_obj->get_name() << " score: " << std::setprecision(std::numeric_limits<time_t>::max_digits10) << e->creationTime << " from ordered set" << dendl;
```

Comment:
why is this needed? didn't we change time format to double for greater precision everywhere?

Reply:
This is in regards to one of Matt's comments about using a time type instead of `double` but I understand your point. Please feel free to discard this change; I will share this with Matt